modules/SofaMiscFem/src/SofaMiscFem/TriangularFEMForceField.inl

```diff
@@ -840,35 +839,25 @@ void TriangularFEMForceField<DataTypes>::computeStress(type::Vec<3,Real> &stress
 // ---	Compute direction of maximum strain (strain = JtD = BD)
 // --------------------------------------------------------------------------------------
 template <class DataTypes>
-void TriangularFEMForceField<DataTypes>::computePrincipalStrain(Index elementIndex, type::Vec<3,Real> &strain )
+void TriangularFEMForceField<DataTypes>::computePrincipalStrain(Index elementIndex, type::Vec<3, Real>& strain)
 {
-    NEWMAT::SymmetricMatrix e(2);
-    e = 0.0;
+    Eigen::Matrix<Real, 2, 2> e;
 
-    NEWMAT::DiagonalMatrix D(2);
-    D = 0.0;
-
-    NEWMAT::Matrix V(2,2);
-    V = 0.0;
-
-    e(1,1) = strain[0];
-    e(1,2) = strain[2];
-    e(2,1) = strain[2];
-    e(2,2) = strain[1];
+    //compute eigenvalues and eigenvectors
+    Eigen::JacobiSVD svd(e, Eigen::ComputeThinU | Eigen::ComputeThinV);
```

Comment:
You don't fill `e` with `strain`?

Reply:
a typo... but not detected because this function is never called
Thanks !